CoreUtilities/src/au/gov/asd/tac/constellation/utilities/gui/NotifyDisplayer.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.utilities.gui;
+
+import org.openide.DialogDisplayer;
+import org.openide.NotifyDescriptor;
+
+/**
+ *
+ * @author aldebaran30701
+ */
+public class NotifyDisplayer {
+
+    /**
+     * Utility display method to show a dialog to the user.
+     * NotifyDescriptor.ERROR_MESSAGE will be used for errors
+     * NotifyDescriptor.INFORMATION_MESSAGE will be used for information
+     * NotifyDescriptor.WARNING_MESSAGE will be used for warnings
+     *
+     * @param message the String message to display on the prompt
+     * @param descriptorType the int value representative of the message type
+     */
+    public static void display(final String message, final int descriptorType) {
```

Comment:
This is great. Can you please mention this in `CHANGELOG.md` as a way to advertise this? Thanks.